<launch>
    <!--<include file="$(find localization)/launch/kidnap_world_moved_pos.launch"/>-->
    <arg name="model" default="$(env TURTLEBOT3_MODEL)" doc="model type [burger, waffle, waffle_pi]"/>
    <arg name="map_file" default="$(find localization)/maps/testHomeMap.yaml"/>
    <arg name="open_rviz" default="true"/>

    <node name="map_server" pkg="map_server" type="map_server" args="$(arg map_file)" respawn="true"/>
    <node pkg="amcl" type="amcl" name="amcl" output="screen">
        <param name="initial_pose_x" value="0.0"/>
        <param name="initial_pose_y" value="0.0"/>
        <param name="initial_pose_a" value="0.0"/>
        <param name="tf-broadcast" value="true"/>
        <param name="first_map_only" value="true" />
    </node>
    <node pkg="localization" type="localize_robot.py" name="localize_robot" output="screen"/>

      <!-- rviz -->
  <group if="$(arg open_rviz)">
    <node pkg="rviz" type="rviz" name="rviz" required="true"
          args="-d $(find localization)/rviz/localization_sim.rviz"/>
  </group>

</launch>